scale/mesos_api/utils.py

```diff
@@ -3,11 +3,107 @@
 from datetime import datetime, timedelta
 
 from django.utils.timezone import utc
+from google.protobuf.internal import enum_type_wrapper
```

Comment:
Yeah, but it was the cleanest way available to me. We might need to rework this if mesos.interface changes. It would be nice if their Python bindings presented a consistent enum/class/something for us to interface with.